python/GafferUI/ColorChooser.py

```diff
@@ -158,23 +158,23 @@ def __init__( self, color=imath.Color3f( 1 ), **kw ) :
 
 		self.__sliders = {}
 		self.__numericWidgets = {}
+		self.__channelLabels = {}
 		self.__componentValueChangedConnections = []
 
 		with self.__column :
```

Comment:
Maybe better to deal with this as part of a future layout change, but perhaps not we have a grid layout, we could use that for everything rather than nest everything in a column?

Reply:
I think I have enough questions about what the layout will become that I'll defer this until later, just to keep the code revision down.